plugins/database/redshift/redshift_test.go

```diff
@@ -290,18 +346,25 @@ func TestPostgreSQL_RevokeUser(t *testing.T) {
 		t.Fatal("Credentials were not revoked")
 	}
 
-	username, password, err = db.CreateUser(context.Background(), statements, usernameConfig, time.Now().Add(2*time.Second))
+	newRequest.Expiration = time.Now().Add(2 * time.Second)
```

Comment:
This is sharing state between test cases. Can this be refactored into a table test instead?

Reply:
Oops, I thought I got rid of all this kind of state sharing. Thanks for catching.